wordpress-mobile/WordPress-iOS
# Quick Start: New visuals for Quick Start Modal

Closes #18552 

## Description
This PR revamps the quick start modal

_PS: This PR is based on #18688 because it depends on the same newly added illustration._

\ | Light Mode | Dark Mode
-|-|-
Know the app tour |![app-light](https://user-images.githubusercontent.com/25306722/169924308-2a4075eb-fe6e-4783-97c8-0ae18899d471.png)|![app-dark](https://user-images.githubusercontent.com/25306722/169924300-66fb56a7-4e96-46aa-ad4b-10ca9fb4588f.png)
Customize tour |![customize-light](https://user-images.githubusercontent.com/25306722/169924323-f98724e2-eaec-4ae1-88f1-6ad41f3e4be6.png)|![customize-dark](https://user-images.githubusercontent.com/25306722/169924317-a379426b-d538-4fab-914b-84f5aa964334.png)
Grow tour |![grow-light](https://user-images.githubusercontent.com/25306722/169924320-7fff42a8-f63e-48a1-b265-48d81bd58b14.png)|![grow-dark](https://user-images.githubusercontent.com/25306722/169924314-21325cda-f437-4146-ae92-789d80e08e39.png)

## Testing Instructions

_Please test the following using both quick start types and on light and dark modes_
1. Enable quick start
2. Tap on the quick start card
3. Make sure the modal is presented and matches the design
4. Tap any task
5. Make sure the modal is dismissed and the task is started
6. Finish the task
7. Open the modal again
8. Make sure the completed task matches the design

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
